identity/src/main/java/com/stripe/android/identity/viewmodel/IdentityViewModel.kt

```diff
@@ -880,7 +880,9 @@ internal class IdentityViewModel constructor(
 
     private fun calculateClearDataParam(dataToBeCollected: CollectedDataParam) =
         ClearDataParam.createFromRequirements(
-            Requirement.values().toMutableSet().minus(
+            requireNotNull(_verificationPage.value?.data?.requirements?.missing) {
+                "VerificationPage not initialized"
+            }.toMutableSet().minus(
```

Comment:
are we fine crashing here?

Reply:
Yes, it's not possible to crash at this point, as `_verificationPage` has to be initialized when the session starts

Reply:
hmm on 2nd thought I think it's safer to provide a fallback and log an error if this happens, updated :)